Add README usage instructions (#1)
nimblehq/publish-github-wiki-action: README.md

  <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png" />
</p>

<h3 align="center">Publish Github Wiki Action</h3>
<p align="center">This action helps syncs your project's documentation with your project's Github Wiki.</p>

-------

## Usage

Add the following workflow as a step in your existing workflow or a new one, replacing `USER_NAME` and `USER_EMAIL` with your account/bot information.

```yml
- name: Publish Github Wiki
  uses: nimblehq/publish-github-wiki-action@v1.0

>   env:

Reviewer: Should we turn these environment variables into inputs?
In other actions, they often pass data to the action via inputs, and we can mark that input required, or set the default value. 
What do you think?
You can check [the example here](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action)

Author: Thanks for the suggestion! I've updated the workflow to pass data via inputs 0c418d9c764702f517d662624c7414d6f0ceaef1
And updated the usage instructions 😄

    USER_NAME: John Smith
    USER_EMAIL: john_smith@email.com
    USER_ACCESS_TOKEN: ${{ secrets.USER_ACCESS_TOKEN }}
```

<!-- TODO: Allow users to customize the wiki directory folder -->
This workflow will synchronize the `.github/wiki` folder with your project's Github Wiki.

## Setup

To use this workflow, your project must have the following:

- The Github Wiki must be enabled and initialized.
- A valid Personal Access Token must be provided to the workflow.

Below are steps to ensure proper setup:

1. **Enable the Wiki feature**

   You can enable the Wiki through your project's "Settings" tab.

   ![Wiki Feature in Settings](https://user-images.githubusercontent.com/34730459/162665798-a6464c67-c74a-427e-9221-4df8310fe9cb.png)

2. **Create at least one Wiki Page in the "Wiki" tab**

>    This is required to initalize the project's Wiki repository.

Reviewer 2: ```suggestion
   This is required to initialize the project's Wiki repository.
```

very well written README 👍

Author: Thanks! Updated in 11592dcf0592b56bf65f197077eb0ef1d8f206b3

   ![Create Wiki Page](https://user-images.githubusercontent.com/34730459/162666191-5f8cf77b-925f-4bec-b3b9-4ae4996444a9.png)

3. **Create a Personal Access Token with the `repo` scope enabled**

   Navigate to the [Personal Access Token](https://github.com/settings/tokens) page under the "Developer Settings" in your **user/bot account** "Settings" and generate the token.

   ![Generate Token](https://user-images.githubusercontent.com/34730459/162668740-b0ef45da-0985-4953-80c7-c60c35f5aefa.png)

   Ensure your token is copied/saved for the next step.

4. **Add the created Personal Access Token as a secret**

   Navigate to "Secrets > Actions" under the projects "Settings" tab to add a secret.

   ![Add new secret](https://user-images.githubusercontent.com/34730459/162669143-6fe258ad-b058-48a1-9c7b-dcbb1ecd0452.png)

## License

> This project is Copyright (c) 2014-2022 Nimble. It is free software,

Reviewer 3: Ditto here (https://github.com/nimblehq/git-template/blob/main/README.md):

```suggestion
This project is Copyright (c) 2014 and onwards Nimble. It is free software
```

and may be redistributed under the terms specified in the [LICENSE] file.

[LICENSE]: /LICENSE

## About

> ![Nimble](https://assets.nimblehq.co/logo/dark/logo-dark-text-160.png)

Reviewer: You can check [this PR](https://github.com/nimblehq/git-template/pull/25) to update this section as well 😄

Author: Nice! Updated in 11592dcf0592b56bf65f197077eb0ef1d8f206b3 😄

>   <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png" />

Reviewer: Should we change this as well?
```suggestion
  <a href="https://nimblehq.co/">
   <picture>
     <source media="(prefers-color-scheme: dark)" srcset="https://assets.nimblehq.co/logo/dark/logo-dark-text-320.png">
     <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png">
   </picture>
 </a>
```
